src/socket/src/sink.rs

```diff
@@ -9,7 +9,7 @@ use async_lock::Mutex;
 use async_lock::MutexGuard;
```

Comment:
bump this crate to `0.8..1` and ensure all dependent create to use this version?